source/includes/User/user.md

```diff
@@ -8,36 +8,122 @@ The real-life person is represented by a Person record in OpenMRS, and a person
+            "systemId": "admin",
+            "userProperties": {
+                "loginAttempts": "0",
+                "emrapi.lastViewedPatientIds": "508,507,311,509,510,511"
+            },
+            "person": {
+                "uuid": "24252571-dd5a-11e6-9d9c-0242ac150002",
+                "display": "Super User",
+                "links": [
+                    {
+                        "rel": "self",
+                        "uri": "http://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrs/ws/rest/v1/person/24252571-dd5a-11e6-9d9c-0242ac150002"
+                    }
+                ]
+            },
+            "privileges": [],
+            "roles": [
+                {
+                    "uuid": "8d94f852-c2cc-11de-8d13-0010c6dffd0f",
+                    "display": "System Developer",
+                    "links": [
+                        {
+                            "rel": "self",
+                            "uri": "http://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrs/ws/rest/v1/role/8d94f852-c2cc-11de-8d13-0010c6dffd0f"
+                        }
+                    ]
+                },
+                {
+                    "uuid": "8d94f280-c2cc-11de-8d13-0010c6dffd0f",
+                    "display": "Provider",
+                    "links": [
+                        {
+                            "rel": "self",
+                            "uri": "http://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrs/ws/rest/v1/role/8d94f280-c2cc-11de-8d13-0010c6dffd0f"
+                        }
+                    ]
+                }
+            ],
+            "retired": false,
+            "links": [
+                {
+                    "rel": "self",
+                    "uri": "http://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrs/ws/rest/v1/user/45ce6c2e-dd5a-11e6-9d9c-0242ac150002"
+                },
+                {
+                    "rel": "full",
+                    "uri": "http://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrshttp://demo.openmrs.org/openmrs/ws/rest/v1/user/45ce6c2e-dd5a-11e6-9d9c-0242ac150002?v=full"
+                }
+            ],
+            "resourceVersion": "1.8"
+        }
+    ]
+}
+
 ```
+
 ```java
 
 OkHttpClient client = new OkHttpClient().newBuilder()
   .build();
 Request request = new Request.Builder()
-  .url("https://demo.openmrs.org/openmrs/ws/rest/v1/user?q=admin")
+  .url("https://demo.openmrs.org/openmrs/ws/rest/v1/user?q=admin&limit=1
+&v=default")
   .method("GET", null)
   .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
-  .addHeader("Cookie", "JSESSIONID=154692F02BBBC664825F3C4C224A474B")
+  .addHeader("Cookie", "JSESSIONID=A2AF09658C73E1ECEC5D3C8C7C249A2D")
   .build();
 Response response = client.newCall(request).execute();
 
 ```
 
+```javascript
+
+var myHeaders = new Headers();
```

Comment:
change this pls.

Reply:
@Akayeshmantha I dint get should I remove the empty line ?